help/en/releasenotes/current-draft-note.shtml

```diff
@@ -572,5 +572,14 @@
         <ul>
             <li>Basic support for TypeScript has been added to the
                 webserver file system.</li>
+            <li>Users that have HighDPI scaling set to something
+                different than 100% can now select whenether or not
+                to force Java to use 100% scaling. It's set in the
+                file <code>settings:JMRI_InitPreferences.ini</code>
```

Comment:
Should this mention the location of this file?

Reply:
I thought that `settings:` would be enough, but maybe it's better to write out `JMRI system preferences directory`.

Is this better?
```
            <li>Users that have HighDPI scaling set to something
                different than 100% can now select whenether or not
                to force Java to use 100% scaling. It's set in the
                file <code>JMRI_InitPreferences.ini</code> in the
                JMRI system preferences directory with the setting
                <code>sun.java2d.uiScale</code>.
                If this value is <code>1</code>, Java forces the
                scaling of JMRI to 100%. If this value is
                <code>0</code>, Java uses the same scaling as the
                system.</li>
```

Reply:
Thanks.  I think that's better for a release note, as it'll help people find it.

Reply:
I have updated the text now.